presto-testng-services/src/main/java/io/prestosql/testng/services/ReportUnannotatedMethods.java

```diff
@@ -50,6 +50,13 @@ public void onBeforeClass(ITestClass testClass)
     private void reportUnannotatedTestMethods(ITestClass testClass)
     {
         Class<?> realClass = testClass.getRealClass();
+
+        if (realClass.getSuperclass() != null &&
```

Comment:
I think it is too simple.

If it helps for now we could leave it.

 Convention tests do not fail because they are convention tests, but because they are using constructions that are problematic. Such construction could happen in the wild without convention tests.

Reply:
Totally agree. This is a stop gap. 

At the same time, inspecting generated class is not very helpful and I did not intend to do this.